PowerEditor/src/WinControls/VerticalFileSwitcher/VerticalFileSwitcherListView.cpp

```diff
@@ -242,7 +242,6 @@ void VerticalFileSwitcherListView::setItemIconStatus(BufferID bufferID)
 			{
 				ListView_SetItemText(_hSelf, i, 1, (LPTSTR)::PathFindExtension(buf->getFileName()));
 			}
-			break;

```

Comment:
I might have changed the `break;` line to a comment, like:

`// do not break here because we could be dealing with a cloned document`

I would do this because the next reader of this source code might think: "This is inefficient, why don't we just break out once we've found the matching buffer" and then they are inclined to put the `break` back in.  :-)

But it is OK the way it is; don't change it.
Wait for acceptance or rejection.